helpers/base.py

```diff
@@ -173,3 +173,25 @@ def ghe_json_message(rec):
         return
 
     return message_rec
+
+def select_key(data, search_key, results):
+    """ Recursively search for a given key and return all values
+    Args:
+        data (dict, list)
+        search_key (string)
+        results (array)
+
+    Returns:
+        (list) all values
+    """
+    if not isinstance(data, (dict, list)):
+        return []
+    if search_key in data:
+        results.append(data[search_key])
```

Comment:
While the above check (`if search_key in data`) will work if this is a list, this line will fail because search_key is not an number and list only support integer indexing.

Maybe we can talk through what this function is going to achieve? Mainly, my concern falls on why we'd be searching lists for a keys (when lists don't have keys)